src/modules/display.js

```diff
@@ -0,0 +1,31 @@
+import addScores from './scores.js';
+import Score from './score.js';
+
+class Display {
+  scores = [];
+
+  localData = 'Score';
+
+  workForm = '';
+
+  constructor() {
+    this.container = document.getElementById('listScore');
+    if (localStorage.getItem(this.localData)) {
+      // eslint-disable-next-line max-len
```

Comment:
- [ ] Please let's avoid disabling linters. Linters can identify certain programming errors earlier on that you may not have noticed while writing code ☺️.

Reply:
> Correction

The `eslint-disable-next-line max-len` error is resolved by splitting the line into three parts

https://github.com/mdg0410/Leaderboard/blob/393f91acdae939a7c38cc03a6ab9183289f1d60a/src/modules/display.js#L13
https://github.com/mdg0410/Leaderboard/blob/393f91acdae939a7c38cc03a6ab9183289f1d60a/src/modules/display.js#L14
https://github.com/mdg0410/Leaderboard/blob/393f91acdae939a7c38cc03a6ab9183289f1d60a/src/modules/display.js#L15